xbmc/input/InertialScrollingHandler.cpp

```diff
@@ -95,10 +98,20 @@ bool CInertialScrollingHandler::CheckForInertialScrolling(const CAction* action)
     auto velocityX = velocitySum.x / m_panPoints.size();
     auto velocityY = velocitySum.y / m_panPoints.size();
 
-    CLog::LogF(LOGDEBUG, "Avg touch velocity: %f,%f up to and including touch at %u ms ago", velocityX, velocityY, m_panPoints.front().TimeElapsed());
+    m_timeToZero = TIME_TO_ZERO_SPEED;
+    auto velocityMax = std::max(std::abs(velocityX), std::abs(velocityY));
+    float dpiScale = CGenericTouchInputHandler::GetInstance().GetScreenDPI() / 160.0f;
 
-    if (std::abs(velocityX) > MINIMUM_SPEED_FOR_INERTIA || std::abs(velocityY) > MINIMUM_SPEED_FOR_INERTIA)
+    CLog::LogF(LOGDEBUG,
```

Comment:
I hesitate when logging input from a source, because we don't have control over not polluting the log. When I see "gesture end", I worry about logging rapid touches.